root/defaults/nginx/site-confs/default.conf.sample

```diff
@@ -12,7 +12,7 @@ server {
     include /config/nginx/ssl.conf;
 
     root /app/www/client/build;
-    index index.html index.htm index.php;
+    index index.html;
```

Comment:
> This specific PR adds `index.htm index.php` as `index` directives.

am i misreading or is it actually removing them?

Reply:
Well now I'm re-looking over this and retracing what I was trying to do a month ago and I think 11 hours ago me wasn't thinking the same thing as 1 month ago me.

11 hours ago me described the inverse of what the PR is actually doing.

The PR removes the index.htm and index.php directives because this image has no php handler and the app build outputs an index.html file to `/app/www/client/build/index.html` so there is no `index.htm`